spec/features/local_login_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Local Login', type: :feature do
+  include LoginHelpers
+
+  context 'when user login is incorrect' do
+    let(:user) { create(:user) }
+
+    it 'shows error banner when login credentials are incorrect' do
+      credentials = { 'user_email' => user.email, 'user_password' => 'bad_pass' }
+      expect { vulcan_sign_in_with('Local Login', credentials) }
```

Comment:
The `User.count` check here doesn't really make sense since local login doesn't ever change the user count. If you want to check something you could see the specific users `sign_in_count` doesn't change.